Animal Wellfare and Euthanasia/SOP_BirdMonitoringAndTracking.md

```diff
@@ -38,6 +38,9 @@ Just as individuals have medical histories, birds in our care also possess medic
 ### Bird Management Platform
 
 To facilitate this tracking, we utilize the [Bird Management Platform](https://prodis.weizmann.ac.il/pls/htmldb/f?p=468:101:11177740050203:::::). Please note that access to this platform is required to view the provided link.
+Both students who are working with the birds, animal fascility workers and the vetrinerian in charge of the animal fascility should have access to this platform. 
```

Comment:
'Both' implies that there are two things but you're listing 3.
Just remove the word 'Both'